### PR TITLE
Add blog post for Gateway API v0.5.0 beta

### PR DESCRIPTION
**Pull request by shaneutt**
This PR adds a blog post for the `v0.5.0` release of [Gateway API](https://github.com/kubernetes-sigs/gateway-api).

### CODE REVIEW

**Review by robscott — COMMENTED** [review]
Thanks @shaneutt! Probably worth mentioning that this is a copy of https://github.com/kubernetes-sigs/gateway-api/pull/1227. If @sftim has time, I'd appreciate their review on this.

/cc @sftim

File: content/en/blog/_posts/2022-06-28-gateway-api-in-beta.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: 'Gateway API First Beta'
+date: 2022-06-28
```
[the remark this answers is not in the document]
Comment (shaneutt) [reply content/en/blog/_posts/2022-06-28-gateway-api-in-beta.md:4]: Sounds good updated in 322b19a6ba9b2b2ac3f25b092a0b172ce870b11f

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: 'Gateway API First Beta'
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:3]:
```suggestion
title: Kubernetes Gateway API Reaches First Beta
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+date: 2022-07-12
+slug: gateway-api-first-beta
+---
+
+**Author:** Shane Utt (Kong)
+
+We're pleased to announce that key Gateway API resources are graduating to
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:10]: Does “we“ mean Kong? If not, please clarify.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+[crd]:https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/
+[concepts]:https://gateway-api.sigs.k8s.io/concepts/api-overview/
+[guides]:https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/
+[impl]:https://gateway-api.sigs.k8s.io/implementations
+[install-crds]:https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/#install-the-crds
+[issue]:https://github.com/kubernetes-sigs/gateway-api/issues/new/choose
+[disc]:https://github.com/kubernetes-sigs/gateway-api/discussions/new
+[community]:https://gateway-api.sigs.k8s.io/contributing/community/
+
+## Release Highlights
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:67]: (nit)
```suggestion
## Release highlights
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+### Graduation to Beta
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:69]:
```suggestion
### Graduation to beta
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+This achievement was marked by the completion of several graduation criteria:
+
+- over a dozen [implementations][impl]
+- the first release of our conformance test suite
+- most of the API surface is actively being used
+- approval from Kubernetes SIG Network API reviewers
+
+To see what's in store for future releases check out the [next steps
+section](/first-beta.md#next-steps).
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:86]: This link doesn't look right.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+[impl]:https://gateway-api.sigs.k8s.io/implementations
+
+### Release Channels
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:90]:
```suggestion
### Release channels
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+and end-users.
+
+For this release we've added the following experimental features:
+
+- [Route select Gateway listeners by port number](https://github.com/kubernetes-sigs/gateway-api/pull/1002)
+- [URL rewrites and path redirects](https://github.com/kubernetes-sigs/gateway-api/pull/945)
+
+[ch]:https://gateway-api.sigs.k8s.io/concepts/versioning/#release-channels-eg-experimental-standard
+
+## Other Improvements
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:118]: (nit)
```suggestion
## Other improvements
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+
+For an exhaustive list of changes included in the `v0.5.0` release, please see
+the [v0.5.0 CHANGELOG entry][changelog].
+
+[changelog]:https://github.com/kubernetes-sigs/gateway-api/blob/master/CHANGELOG.md#v050
+
+## Next Steps
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:125]: (nit)
```suggestion
## Next Steps
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+title: 'Gateway API First Beta'
+date: 2022-07-12
+slug: gateway-api-first-beta
+---
+
+**Author:** Shane Utt (Kong)
+
+We're pleased to announce that key Gateway API resources are graduating to
+beta with the `v0.5.0` release of the API. In addition to `GatewayClass`,
+`Gateway` and `HTTPRoute` graduating to beta, this release introduces several
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:12]: I'd make GatewayClass, Gateway and HTTPRoute be hyperlinks to https://gateway-api.sigs.k8s.io/beta1/api-types/gatewayclass/ and so on (remember to produce those docs for the release, too).

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+new concepts to the API that we'll cover below.
+
+## What is Gateway API
+
+Gateway API is a collection of resources centered around `Gateway` resources
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:17]: I'd make Gateway be a hyperlink to https://gateway-api.sigs.k8s.io/beta1/api-types/gateway/

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+(which represent the underlying network gateways / proxy servers) to enable
+robust Kubernetes service networking through expressive, extensible and
+role-oriented interfaces that are implemented by many vendors and have broad
+industry support.
+
+Originally conceived as a successor to the well known `Ingress` API, the
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:23]: I'd make Ingress be a hyperlink to https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+this release there are over a dozen [implementations][impl] for popular API
+gateways and service meshes and guides are available to start exploring quickly.
+
+[impl]:https://gateway-api.sigs.k8s.io/implementations
+
+### Getting Started
+
+The Gateway API project itself is a Kubernetes API like `Ingress` and you can
+think of the implementations of it as similar to the implementations for
+`IngressClass` or `StorageClass` in that Kubernetes itself has no default
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:41]: I'd make IngressClass and StorageClass be hyperlinks.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 14]
```diff
@@ -0,0 +1,142 @@
...
+[changelog]:https://github.com/kubernetes-sigs/gateway-api/blob/master/CHANGELOG.md#v050
+
+## Next Steps
+
+As we continue to mature the API for production use cases, the following is a
+(non-exhaustive) list of some of the highlights for upcoming releases:
+
+- [GRPCRoute][gep1016] for [gRPC][grpc] traffic routing
+- [route delegation][pr1085]
+- Layer 4 API maturity: `v1beta1` for `TCPRoute`, `UDPRoute` and `TLSRoute`
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:132]: I'd make TCPRoute, UDPRoute and TLSRoute be hyperlinks.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 15]
```diff
@@ -0,0 +1,142 @@
...
+- Expanded support and coverage for service mesh use cases (to be discussed in Working Group Mesh)
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]: _Is_ there a WG Mesh? If so, consider updating https://github.com/kubernetes/community/tree/master/sig-network to mention it.

If that's more of an informal working group, maybe reword to avoid implying that there's a project-wide Kubernetes working group for mesh.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 16]
```diff
@@ -0,0 +1,142 @@
...
+robust Kubernetes service networking through expressive, extensible and
+role-oriented interfaces that are implemented by many vendors and have broad
+industry support.
+
+Originally conceived as a successor to the well known `Ingress` API, the
+benefits of Gateway API include (but are not limited to) explicit support for
+many commonly used networking protocols (e.g. `HTTP`, `TLS`, `TCP`, `UDP`) as
+well as tightly integrated support for Transport Layer Security (TLS). The
+`Gateway` resource in particular enables implementations to manage the lifecycle
+of network gateways as a native Kubernetes API.
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:28]: Gateway is _not_ a native Kubernetes API. It's an official extension API (or it might be semi-official - see following questions).

- Does Gateway and friends go through formal API review? If it does, that review is a benefit to cluster operators who want to rely on it.
- Similarly, does Gateway now benefit from following the official [Kubernetes deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/)? Whatever the answer is, we should make that clear in the graduation announcement.

**Review by sftim — COMMENTED** [review]
Overall feedback:

- This is short notice for a blog article; 2 working days does not allow much time for editing especially as I see this is not yet marked ready for review.
  - If the 12th of July date is important, I suggest asking folk in #sig-docs-blog to help get this out on time.
  - We (SIG Docs blog team) can tweak the details of the article after publication if this is time sensitive. Next time, please consider submitting earlier.
  - I've added some early feedback.
- I recommend adding a link to https://github.com/kubernetes/community/tree/master/sig-network#gateway-api somewhere in the page.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 17]
```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: 'Gateway API First Beta'
+date: 2022-07-12
+slug: gateway-api-first-beta
+---
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:6]:
```suggestion
canonicalUrl: https://gateway-api.sigs.k8s.io/blog/2022/introducing-v1beta1/
---
```
(that is to follow the pattern from https://gateway-api.sigs.k8s.io/blog/2021/introducing-v1alpha2/ - and I therefore also suggest changing the URL of the draft article on the Gateway API site)

**Comment by sftim** [comment]
/retitle Add blog post for Gateway API v0.5.0 beta

(for this repo, “docs” is implied)

**Comment by sftim** [comment]
/hold

This blog is announcing features from a KEP that has not been accepted.

I have two related opinions:
- given the target audience and early state for this feature, the article is a better fit for https://kubernetes.dev/blog/
- we should not publish the article text as-is

**Comment by sftim** [comment]
/hold cancel
Sorry

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 11 above)
Comment (divya-mohan0209) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:17]: @sftim the link you provided somehow runs into a 404 for me. I'm assuming you mean this link - https://gateway-api.sigs.k8s.io/api-types/gateway/ or is there something that is supposed to be there but isn't?

Comment (sftim) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:17]: Hmm. yes. I thought I saw a link to https://gateway-api.sigs.k8s.io/v1alpha1/api-types/gateway/ but that doesn't seem to work when I check now.

Linking to https://gateway-api.sigs.k8s.io/api-types/gateway/ seems right.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 16 above)
Comment (sftim) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:28]: :thought_balloon: if Gateway and friends are an official, API-reviewed Kubernetes API, then for beta onwards we can also document that API within https://kubernetes.io/docs/

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 15 above)
Comment (craigbox) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]: We proposed one but the Steering Committee in their wisdom decided it would be best not to set up such.  I'll be proposing some edits in both branches to fix this.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 3 above)
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:10]:
```suggestion
Kubernetes SIG Networking is pleased to announce that key Gateway API resources are graduating to
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 6 above)
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:86]:
```suggestion
To see what's in store for future releases check out the next steps section.
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 10 above)
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:12]:
```suggestion
beta with the `v0.5.0` release of the API. In addition to [GatewayClass](https://gateway-api.sigs.k8s.io/api-types/gatewayclass/),
[Gateway](https://gateway-api.sigs.k8s.io/api-types/gateway/) and [HTTPRoute](https://gateway-api.sigs.k8s.io/api-types/httproute/) graduating to beta, this release introduces several
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 11 above)
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:17]:
```suggestion
Gateway API is a collection of resources centered around [Gateway](https://gateway-api.sigs.k8s.io/api-types/gateway/) resources
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 12 above)
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:23]:
```suggestion
Originally conceived as a successor to the well known [Ingress](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/) API, the
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 16 above)
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:28]: For now just so that we don't get caught up on this I'll just remove this language:
```suggestion
of network gateways as a Kubernetes API.
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 13 above)
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:41]:
```suggestion
[IngressClass](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class) or [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) in that Kubernetes itself has no default
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 9 above)
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:125]:
```suggestion
## Next steps
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 14 above)
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:132]:
```suggestion
- Layer 4 API maturity: `v1beta1` for [TCPRoute](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha2/tcproute_types.go), [UDPRoute](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha2/udproute_types.go) and [TLSRoute](https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha2/tlsroute_types.go)
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 15 above)
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]:
```suggestion
- Expanded support and coverage for service mesh use cases
```

**Comment by shaneutt** [comment]
@sftim I believe I have accounted for all your comments, please let me know if there's anything lingering you wish me to take care of :bow:

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 18]
```diff
@@ -0,0 +1,154 @@
...
+
+- `GatewayClass`
+- `Gateway`
+- `HTTPRoute`
+
+This achievement was marked by the completion of several graduation criteria:
+
+- over a dozen [implementations][impl]
+- the first release of our conformance test suite
+- most of the API surface is actively being used
```
Comment (robscott) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:91]:
```suggestion
- Most of the API surface is actively being used.
```

**Review by robscott — COMMENTED** [review]
Thanks for all the work on this @shaneutt! I've added a bunch of non-blocking nits, feel free to ignore if they don't make sense.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 19]
```diff
@@ -0,0 +1,154 @@
...
+## Next steps
+
+As we continue to mature the API for production use cases, the following is a
+(non-exhaustive) list of some of the highlights for upcoming releases:
+
+- [GRPCRoute][gep1016] for [gRPC][grpc] traffic routing
+- [route delegation][pr1085]
+- Layer 4 API maturity: `v1beta1` for [TCPRoute][tcpr], [UDPRoute][udpr] and
+  [TLSRoute][tlsr]
+- Expanded support and coverage for service mesh use cases
```
[the remark this answers is not in the document]
Comment (craigbox) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:142]: I'm planning on adding a paragraph ahead of this one to announce the new mesh initiative and socialise the working group.  Just waiting on sign-off on what we're called!

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 20]
```diff
@@ -0,0 +1,154 @@
...
+Kubernetes SIG Networking is pleased to announce that key Gateway API resources
+are graduating to beta with the `v0.5.0` release of the API. In addition to
+[GatewayClass][gwc], [Gateway][gw] and [HTTPRoute][httpr] graduating to beta,
+this release introduces several new concepts to the API that we'll cover below.
+
+[gwc]:https://gateway-api.sigs.k8s.io/api-types/gatewayclass/
+[gw]:https://gateway-api.sigs.k8s.io/api-types/gateway/
+[httpr]:https://gateway-api.sigs.k8s.io/api-types/httproute/
+
+## What is Gateway API
```
[the remark this answers is not in the document]
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:20]: I guess it's more of a personal preference thing. My personal feelings is that it's not necessary to use the space for this: anyone who's interested in diving deeper can follow any of the links to our site and find the diagrams and more. If you feel strongly that we should copy our main page diagram(s) to this section let me know and I'll add it.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 21]
```diff
@@ -0,0 +1,154 @@
...
+The `v0.5.0` release is particularly historic because it marks the growth in
+maturity to a `v1beta1` release for some of our key APIs:
+
+- `GatewayClass`
+- `Gateway`
+- `HTTPRoute`
+
+This achievement was marked by the completion of several graduation criteria:
+
+- over a dozen [implementations][impl]
```
[the remark this answers is not in the document]
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:89]:
```suggestion
- API has been [widely implemented][impl].
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 22]
```diff
@@ -0,0 +1,154 @@
...
+### Graduation to beta
+
+The `v0.5.0` release is particularly historic because it marks the growth in
+maturity to a `v1beta1` release for some of our key APIs:
+
+- `GatewayClass`
+- `Gateway`
+- `HTTPRoute`
+
+This achievement was marked by the completion of several graduation criteria:
```
[the remark this answers is not in the document]
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:87]: :+1:

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 23]
```diff
@@ -0,0 +1,154 @@
...
+- un-graduated fields that are considered experimental
+
+Release channels are used internally to enable iterative development with
+quick turnaround, and externally to indicate feature stability to implementors
+and end-users.
+
+For this release we've added the following experimental features:
+
+- [Route select Gateway listeners by port number](https://github.com/kubernetes-sigs/gateway-api/pull/1002)
+- [URL rewrites and path redirects](https://github.com/kubernetes-sigs/gateway-api/pull/945)
```
[the remark this answers is not in the document]
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:122]:
```suggestion
- [URL rewrites and path redirects](https://gateway-api.sigs.k8s.io/geps/gep-726/)
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 24]
```diff
@@ -0,0 +1,154 @@
...
+
+[ch]:https://gateway-api.sigs.k8s.io/concepts/versioning/#release-channels-eg-experimental-standard
+
+## Other improvements
+
+For an exhaustive list of changes included in the `v0.5.0` release, please see
+the [v0.5.0 CHANGELOG entry][changelog].
```
[the remark this answers is not in the document]
Comment (shaneutt) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:129]:
```suggestion
the [v0.5.0 release page][releasepage].
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 25]
```diff
@@ -0,0 +1,154 @@
...
+
+[changelog]:https://github.com/kubernetes-sigs/gateway-api/blob/master/CHANGELOG.md#v050
```
Comment (shaneutt) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:131]:
```suggestion
[releasepage]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.4.3
```

**Comment by sftim** [comment]
> Once approved I will feed the updates from this PR back to the Gateway API PR.

We won't _actually_ approve this until the other PR is ready to go. We can provide an informal approval to let you know that changes here are OK to copy upstream.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 26]
```diff
@@ -0,0 +1,153 @@
...
+[ch]:https://gateway-api.sigs.k8s.io/concepts/versioning/#release-channels-eg-experimental-standard
+
+## Other improvements
+
+For an exhaustive list of changes included in the `v0.5.0` release, please see
+the [v0.5.0 release page][releasepage].
+
+[releasepage]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.4.3
+
+## Next steps
```
Comment (craigbox) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]:
```suggestion
## Gateway API for service mesh 

Some service mesh projects have [already implemented support for the Gateway API](https://gateway-api.sigs.k8s.io/implementations/). Significant overlap between the Service Mesh Interface (SMI) APIs and the Gateway API has [inspired discussion in the SMI community](https://github.com/servicemeshinterface/smi-spec/issues/249) about possible integration.

We are pleased to announce that the service mesh community, including representatives from Cilium Service Mesh, Consul, Istio, Kuma, Linkerd, NGINX Service Mesh and Open Service Mesh, is coming together in a new group for [Gateway API Mesh Management and Administration](https://docs.google.com/document/d/1Qt3N1RLnRom7jKNhcnNLg_xaPn0TwKGuxxgXLOguNvE/edit#).

This group will deliver [enhancement proposals](https://gateway-api.sigs.k8s.io/v1beta1/contributing/gep/) consisting of resources, additions, and modifications to the Gateway API specification for mesh and mesh-adjacent use-cases.

This work has begun with [an exploration of using Gateway API for east/west traffic](https://docs.google.com/document/d/1T_DtMQoq2tccLAtJTpo3c0ohjm25vRS35MsestSL9QU/edit#heading=h.jt37re3yi6k5) and will continue with enhancement in areas such as authentication and authorization policy. 

## Next Steps
```

Comment (craigbox) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]: @keithmattix @howardjohn @mikemorris @robscott PTAL

Comment (craigbox) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]: done

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 27]
```diff
@@ -0,0 +1,153 @@
...
+- [URL rewrites and path redirects](https://gateway-api.sigs.k8s.io/geps/gep-726/)
+
+[ch]:https://gateway-api.sigs.k8s.io/concepts/versioning/#release-channels-eg-experimental-standard
+
+## Other improvements
+
+For an exhaustive list of changes included in the `v0.5.0` release, please see
+the [v0.5.0 release page][releasepage].
+
+[releasepage]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.4.3
```
Comment (sftim) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:131]:
```suggestion
[releasepage]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.5.0
```
BTW for this site we prefer inline links rather than the footnote style, but that's a preference not a merge blocker.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 26 above)
Comment (howardjohn) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]: LGTM. If we want GAMMA to be an acronym we may want to expand it, but I don't mind either way

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 28]
```diff
@@ -0,0 +1,153 @@
...
+Originally conceived as a successor to the well known [Ingress][ing] API, the
+benefits of Gateway API include (but are not limited to) explicit support for
+many commonly used networking protocols (e.g. `HTTP`, `TLS`, `TCP`, `UDP`) as
+well as tightly integrated support for Transport Layer Security (TLS). The
+`Gateway` resource in particular enables implementations to manage the lifecycle
+of network gateways as a Kubernetes API.
+
+If you're an end-user interested in some of the benefits of Gateway API we
+invite you to jump in and find an implementation that suits you. At the time of
+this release there are over a dozen [implementations][impl] for popular API
```
Comment (howardjohn) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:37]: Should we highlight integrations as well? I think we have flagger and cert-manager. It shows the widespread adoption pretty well I think

**Review by howardjohn — COMMENTED** [review]
LGTM

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 26 above)
Comment (craigbox) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]: We do, later in the sentence!  Can edit around that if it's non-obvious; I tried to say explicitly it was a workstream but the sentence was super long and contained "Gateway API" three times

Comment (keithmattix) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]: Maybe bold the first letter of the expanded name for clarity?

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 29]
```diff
@@ -0,0 +1,153 @@
...
+
+- [GRPCRoute][gep1016] for [gRPC][grpc] traffic routing
+- [Route delegation][pr1085]
+- Layer 4 API maturity: `v1beta1` for [TCPRoute][tcpr], [UDPRoute][udpr] and
+  [TLSRoute][tlsr]
+- Expanded support and coverage for service mesh use cases
+
+If there's something on this list you want to get involved in, or there's
+something not on this list that you want to advocate for to get on the roadmap
+please join us in the #sig-network-gateway-api channel on Kubernetes Slack or our weekly [community calls](https://gateway-api.sigs.k8s.io/contributing/community/#meetings)
```
Comment (craigbox) [inline content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:145]:
```suggestion
please join us in the #sig-network-gateway-api channel on Kubernetes Slack or our weekly [community calls](https://gateway-api.sigs.k8s.io/contributing/community/#meetings).
```

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 26 above)
Comment (craigbox) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]: done

**Comment by robscott** [comment]
We've updated the Gateway API release to tomorrow (July 13), Shane's OOO today so I'm going to fork this PR to finish up the last few changes.

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md [hunk 30]
```diff
@@ -0,0 +1,153 @@
...
+`Gateway` resource in particular enables implementations to manage the lifecycle
+of network gateways as a Kubernetes API.
+
+If you're an end-user interested in some of the benefits of Gateway API we
+invite you to jump in and find an implementation that suits you. At the time of
+this release there are over a dozen [implementations][impl] for popular API
+gateways and service meshes and guides are available to start exploring quickly.
+
+[gw]:https://gateway-api.sigs.k8s.io/api-types/gateway/
+[ing]:https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/)
```
[the remark this answers is not in the document]
Comment (robscott) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:41]: Fixed in #34968

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 28 above)
Comment (robscott) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:37]: I think it's OK since the linked implementations page also includes integrations, but open to ways to more clearly reference those here as well

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 29 above)
Comment (robscott) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:145]: Fixed in #34968

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 26 above)
Comment (robscott) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:133]: Added in #34968

File: content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md (hunk 20 above)
[the remark this answers is not in the document]
Comment (robscott) [reply content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md:20]: Yeah I'm fine with that, will leave this out

**Comment by sftim** [comment]
I've also reviewed #34968.

BTW, https://github.com/kubernetes/website/pull/34884#issuecomment-1182540944 was the first notice I had to confirm the intended release date. With the staffing of the blog team being what it is, clearer comms here would have helped.

**Comment by robscott** [comment]
Closing this one in favor of #34968 to avoid any confusion.

/close
